draft-ietf-rats-architecture.md

```diff
@@ -1149,8 +1149,7 @@ or might be defined relative to some other timestamp or timeticks counter.
 
 | ID | Event                       | Explanation of event
 |----|-----------------------------|-----------------------
-| VG | Value generation            | A value to appear in a Claim was created.
-| AA | Attester awareness          | An Attesting Environment starts to be aware of a new/changed Claim value.
+| VC | Value created               | A value to appear in a Claim was created.  In some cases, a value may have technically existed before an Attester became aware of it but the Attester might have no idea how long it has had that value.  In such a case, the Value created time is the time at which the Claim containing the copy of the value was created.
```

Comment:
```suggestion
| VG | Value generated             | A value to appear in a Claim was created.  In some cases, a value may have technically existed before an Attester became aware of it but the Attester might have no idea how long it has had that value.  In such a case, the Value created time is the time at which the Claim containing the copy of the value was created.
```

Reply:
Less growing pains, more consistence?

Reply:
The underlying question here is. Is it necessary that some Verifiers are informed about the "timeliness" of the "new VG"?

As Dave already highlighted, we discussed this a bit and the sanity check ask is the following:

If we collapse VG and AA semantics, it is not distinguishable for Verifier, if a corresponding Value generated time is about Attester awareness/Claim generation or Value Generation. Is that okay or does that create uncertainty that could result in degradation of evidence trustworthiness?